Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16
```

Comment:
is it a way to bind this to the latest lts version?

Reply:
thank you for review, fixed it.

Reply:
it should to point to latest LTS (right now is v14)